content/en/docs/reference/_index.md

```diff
@@ -20,6 +20,7 @@ This section of the Kubernetes documentation contains references.
 
 * [Kubernetes API Overview](/docs/reference/using-api/api-overview/) - Overview of the API for Kubernetes.
 * [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
+* [Kubernetes API Reference for Prior Verions](/docs/reference/kubernetes-api/)
```

Comment:
@tengqm , Some more feedback:
Not sure about repeating the `API Reference` heading again.
From the left hand tree, there is `Reference` and then the sub folder, `API References` containing the 'taints' page and the links page?
The new page, `API References` repeats the link to the current API reference and adds the 
links to the previous four versions of the API reference. These links have to be maintained?
Would suggest changing,
`Kubernetes API Reference v1.18` to look like the other links. Possibly add (latest)?
At some point, I could see tying in the API reference links with the `Versions` drop down menu.

nit: typo: s/Verions/Versions/